src/tqec/circuit/observable_qubits.py

```diff
@@ -0,0 +1,96 @@
+from typing import Any, Mapping, Sequence
```

Comment:
```suggestion
from __future__ import annotations

from typing import Any, Mapping, Sequence
```
fixes some test issues.